library/network/src/modules/SuSEFirewall.rb

```diff
@@ -154,9 +154,64 @@ def self.create
   # ----------------------------------------------------------------------------
   # SuSEFirewalld Class. Trying to provide relevent pieces of SF2 functionality via
   # firewalld.
-  class SuSEFirewalld < Module
+  class SuSEFirewalld < Firewall
+    attr_reader :special_all_interface_zone
+
+    # Valid attributes for firewalld zones
```

Comment:
Do  I understand the types of the attributes right?
interfaces: Array(String)
masquerade: Boolean
modified: Boolean
ports: Array(Integer) or of String? [Range](http://ruby-doc.org/core-2.1.0/Range.html)?
protocols: Array(String)
services: Array(String)